sqlglot/dialects/mysql.py

```diff
@@ -580,6 +587,21 @@ class Generator(generator.Generator):
             exp.DataType.Type.VARCHAR: "CHAR",
         }
 
+        UNSIGNED_TYPES = {
+            exp.DataType.Type.UBIGINT,
+            exp.DataType.Type.UINT,
+            exp.DataType.Type.UMEDIUMINT,
+            exp.DataType.Type.USMALLINT,
+            exp.DataType.Type.UTINYINT,
+        }
```

Comment:
Let me see if I can dry this up a bit with a dict that I can reuse in `TYPE_MAPPING`...